src/App.scss

```diff
@@ -1,3 +1,7 @@
 iframe {
   display: none;
```

Comment:
Hiding all iframes using `display: none;` will make them completely invisible and non-interactive. Ensure this is the desired effect, as it will prevent any iframe content from being displayed or interacted with.